test/e2e/testing-manifests/storage-csi/cephfs/node_ds.yaml

```diff
@@ -0,0 +1,84 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-cephfs-node
+spec:
+  selector:
+    matchLabels:
+      app: csi-cephfs-driver
+  serviceName: csi-cephfs
+  template:
+    metadata:
+      labels:
+        app: csi-cephfs-driver
+    spec:
+      serviceAccount: cephfs-node
+      containers:
+        - name: csi-driver-registrar
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/driver-registrar:v0.2.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/kubelet/plugins/csi-cephfsplugin/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /var/lib/kubelet/plugins/csi-cephfsplugin
+        - name: cephfs-driver
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          imagePullPolicy: IfNotPresent
+          image: quay.io/cephcsi/cephfsplugin:v0.3.0
```

Comment:
Sorry for responding late! You're using csi-cephfs 0.3.0 here, but the external-provisioner, external-attacher and driver-registrar are at 0.2.0 - you should have all those at `v0.3.0` (AFAIK external-provisioner populates node stage secrets reference from 0.3.0+).